app/models/article.rb

```diff
@@ -30,17 +32,18 @@ class Article < ActiveRecord::Base
   has_many :articles_courses, class_name: ArticlesCourses
   has_many :courses, -> { uniq }, through: :articles_courses
   has_many :assignments
+  belongs_to :wiki
+
+  # Allows us to directly #update from the Tools response
+  alias_attribute :page_id, :native_id
```

Comment:
Great advice, I'll change the field name in the importer.  More descriptive field names will be an improvement, and will clarify code that consumes these records.